CONTRIBUTING.md

```diff
@@ -8,24 +8,35 @@
   - remove the wrapping conditions for `installCRDs` from all CRDs yaml files (will remove it from `values.yaml` too)
     - IMPORTANT NOTE: keep it for the Cluster* CRDs, just align the conditions to only check `crds.createClusterExternalSecret`
       and `crds.createClusterSecretStore` respectively
-- helper templates
-  - copy over upstream `_helpers.tpl` as `_eso_helpers.tpl`
-- generic templates
-  - copy over templates
-- values.yaml
-  - remove `installCRDs: true`, it will be controlled by `giantswarm.crds.install` instead
-  - remove `crds` section, we install all resources with a few exceptions
-    - IMPORTANT NOTE: keep `crds.createClusterExternalSecret`
-    - IMPORTANT NOTE: keep `crds.createClusterSecretStore`
-  - moved `.image` under `.giantswarm.images` + remove `.image`
-  - moved `.certController.image` under `.giantswarm.images` + remove `.certController.image`
-  - moved `.webhook.image` under `.giantswarm.images` + remove `.webhook.image`
-  - same for all 3 `securityContext` and `resources`
-  - check / reset all the above fields to the Giant Swarm image, security context, resources, etc. in the templates
-    including, but not limited to:
-    - `helm/external-secrets/templates/deployment.yaml`
-    - `helm/external-secrets/templates/cert-controller-deployment.yaml`
-    - `helm/external-secrets/templates/webhook-deployment.yaml`
+- sync subtree with upstream (based on: https://handbook.giantswarm.io/product/managed-apps/dev-experience/git-subtree/)
+  - pull changes to https://github.com/giantswarm/external-secrets-copy
+    - create and push a branch from the target tag you want to upgrade to e.g. from `v0.7.0` as `tag-v0-7-0`
+  - ```bash
+    # If you don't already have it:
+    git remote add -f --no-tags upstream-copy git@github.com:giantswarm/external-secrets-copy.git
+    
+    git subtree add --prefix upstream-tmp upstream-copy tag-v0-7-0
+    git subtree split -P upstream-tmp/deploy/charts/external-secrets -b tmp-split-branch
+    
+    # Prepare upgrade branch
+    git checkout main
+    git pull
+    git checkout -b upgrade-to-v0-7-0
+   
+    # Add subtree (if it does not exist yet) 
```

Comment:
In my docs I skipped this optional `add` here - it needs to be done only once and I did already, so in every other case it will be `subtree merge`